app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any
+from copy import deepcopy
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+    ) -> None:
+        self._capacity = 8
+        self._load_factor = 2 / 3
+        self._threshold = self._capacity * self._load_factor
+        self._hash_table = [[] for _ in range(self._capacity)]
```

Comment:
So this suggests that you create a separate data structure `Node` that has these attributes `key, hash, value`. If you do so your code will be better structured

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.

Reply:
But there is a custom dictionary and I did not wanna create Nodes deliberately. And It suggests that I need to change all logic 'cause It won't be this method of cell storing. Do I really need to change all dictionary?

Reply:
Usually better to ask your question in a chat because mentors don't check github 
And yes, better to implement Node class